templates/parachain/README.md

```diff
@@ -63,14 +63,9 @@ You still need a relaychain node - you can download the `polkadot`
 (and the accompanying `polkadot-prepare-worker` and `polkadot-execute-worker`)
 binaries from [Polkadot SDK releases](https://github.com/paritytech/polkadot-sdk/releases/latest).
```

Comment:
Download binaries (of polkadot) only works on linux (x86_64), we should add a comment to compile those in macOS or arm64.

Reply:
I prefer we solve like this: https://github.com/paritytech/polkadot-sdk/issues/802

Reply:
Totally agree, we should push to include those as part of the release process to improve dx.